### ISSUE TITLE
mysql never cleans

### ISSUE THREAD

**Issue by knappe** [emerleite/node-database-cleaner#25]
It sure looks like no one has ever used the mysql cleaner.

```
        if (table['Tables_in_database_cleaner'] != 'schema_migrations') {
          db.query("DELETE FROM " + table['Tables_in_database_cleaner'], function() {
```
From
https://github.com/emerleite/node-database-cleaner/blob/5b34ab172f7bc11a4d7561d2784c8d7301a4a7e3/lib/database-cleaner.js#L39

I'm not really sure what this is supposed to be doing.  It appears like we should be comparing `table['Table_in_<db_name>']` here, not `table['Tables_in_database_cleaner']`.

**Comment by ScotterC** [comment]
:+1: Confirmed locally.  @knappe can you PR your branch over? It works well for me

**Comment by emerleite** [comment]
Just released 0.9.2 with this fix.
